src/main/java/com/aws/greengrass/security/SecurityService.java

```diff
@@ -110,7 +110,7 @@ public void registerCryptoKeyProvider(CryptoKeySpi keyProvider) throws ServicePr
      */
     public void registerMqttConnectionProvider(MqttConnectionSpi mqttProvider) throws ServiceProviderConflictException {
         CaseInsensitiveString keyType = new CaseInsensitiveString(mqttProvider.supportedKeyType());
-        logger.atInfo().kv(KEY_TYPE, keyType).log("Register crypto key service provider");
+        logger.atInfo().kv(KEY_TYPE, keyType).log("Register MQTT connection provider");
```

Comment:
```suggestion
        logger.atInfo().kv(KEY_TYPE, keyType).log("Register MQTT connection security provider");
```

Reply:
@MikeDombo , happy to do so, but crypto key uses "service" rather than "security". So yeah, definitely "security"?

Reply:
Yes, in this case though it isn't providing an mqtt connection, rather it is providing the pluggable security for the mqtt connection. Not that it matters all that much... but I think my suggestion is still clear and accurate.

Reply:
OK done